File: src/split-char.java
/*
#	What Is This: programming samples
#	Author: Makoto Takeshita <takeshita.sample@gmail.com>
#	URL: http://simplesandsamples.com
#	Version: UNBORN
#	
#	Usage:
#	 1. git clone https://github.com/takeshitamakoto/sss.git
#	 2. change the directory name to easy-to-use name. (e.g. sss -> sample)
#	 3. open sss/src/filename when you need any help.
#	
*/

import java.io.*;
class Test{
	public static void main(String args[]){
		String str="HELLO";
		String[] dst=str.split("");
		for(int i=1;i<dst.length;i++){
			System.out.println("["+dst[i]+"]");
		}
	}
}

File: src/split-char.vb
'
'	What Is This: programming samples
'	Author: Makoto Takeshita <takeshita.sample@gmail.com>
'	URL: http://simplesandsamples.com
'	Version: UNBORN
'	
'	Usage:
'	 1. git clone https://github.com/takeshitamakoto/sss.git
'	 2. change the directory name to easy-to-use name. (e.g. sss -> sample)
'	 3. open sss/src/filename when you need any help . 
'	

Imports System	'for System.Console.WriteLine

Class Test
	Shared Sub Main()
		Dim s As String = "HELLO"
		Dim a As Char() = s.ToCharArray()
		For Each m As Char In a
			Console.WriteLine("[{0}]",m)
		Next
	End Sub
End Class

